.hintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "connector": {
```

Comment:
> This is just an example of how to add inline comments 

- This file should be added to the .gitignore as is only needed to check Linters locally.

Reply:
Thanks a lot for your kind and inspiring review Euqueme, I really appreciate it :).
Also thanks for your tips and suggestions, and for being so nice that you even help me with examples. 
Let's see how everything evolves, and in the future, I will consider becoming a code reviewer so I can also inspire and help others as you did with me.
Thanks and best regards :)